maps/cogmap.dmm

```diff
@@ -27011,15 +27011,15 @@
 /turf/simulated/floor/grime,
 /area/station/maintenance/disposal)
 "bgD" = (
-/obj/item/storage/box/evidence,
-/obj/item/storage/box/evidence,
-/obj/item/storage/box/evidence,
-/obj/item/storage/box/evidence,
 /obj/decal/cleanable/dirt,
 /obj/disposalpipe/segment/ejection{
 	dir = 4
 	},
 /obj/decal/tile_edge/stripe/extra_big,
+/obj/item/storage/box/evidence,
+/obj/item/storage/box/evidence,
+/obj/item/storage/box/evidence,
+/obj/item/storage/box/evidence,
```

Comment:
looks like automatic changes from a map editor, but you didnt make any other changes